blacklight-spotlight.gemspec

```diff
@@ -56,7 +56,7 @@ these collections.)
   s.add_development_dependency 'rspec-activemodel-mocks'
   s.add_development_dependency 'rspec-collection_matchers'
   s.add_development_dependency 'capybara', '>= 2.5.0'
-  s.add_development_dependency 'rubocop'
+  s.add_development_dependency 'rubocop', '~> 0.35'
```

Comment:
Let me rephrase the commit message that changed it. The `AbcSize` is really incidental, it's really about making sure the team and travis are in sync with these kinds of upstream changes (or, I was annoyed that travis was failing a build that worked locally).